newsfragments/2762.misc.rst

```diff
@@ -0,0 +1 @@
+Stop writing bytes to log file which causes exceptions - instead write the hex representation.
```

Comment:
IMHO should be a `bugfix` even if it doesn't cause a breaking error / causes a silent error.

Reply:
I was on the fence about it - my thought was that someone may freak out over seeing the exception and the logs would not contain the intended entry.

Are you voting to remove the newsfragment?

Reply:
Newsfragment is great, I just wanted to share that `newsfragments/2762.bugfix.rst` sits better with me.

Reply:
✔️